src/components/landing/pages/profile/settings/profile-account-management.tsx

```diff
@@ -0,0 +1,72 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { Fragment, useState } from 'react'
+import { Button, Card, Modal } from 'react-bootstrap'
+import { Trans, useTranslation } from 'react-i18next'
+import { doUserDeletion } from '../../../../../api/user'
+import { clearUser } from '../../../../../redux/user/methods'
+import { getBackendUrl } from '../../../../../utils/apiUtils'
+
+export const ProfileAccountManagement: React.FC = () => {
+  const { t } = useTranslation()
+  const [showDeleteModal, setShowDeleteModal] = useState(false)
+  const [deletionButtonText, setDeletionButtonText] = useState('')
+  const [deletionButtonActive, setDeletionButtonActive] = useState(false)
+
+  const handleModalClose = () => {
+    setShowDeleteModal(false)
+  }
+
+  const doCountdown = (value: number) => {
+    if (value === 0) {
```

Comment:
You should cancel the countdown if the modal was closed. If you open it, then close it and open it again, then two countdowns are batteling.